src/components/Login/Login.js

```diff
@@ -0,0 +1,119 @@
+import { useState, useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { useNavigate } from "react-router-dom";
+import "./login.css";
+import { login } from "../../redux/login/actions/login";
+
+const LoginPage = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const successOrErrorMessage = useSelector((state) => state.login);
+  const [emailError, setEmailError] = useState("");
+  const [passwordError, setPasswordError] = useState("");
+  const passwordRegex =
+    /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])[a-zA-Z0-9!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?]{8,}$/;
+  const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+  const validateForm = (event) => {
+    event.preventDefault();
+    const email = document.getElementById("email").value;
```

Comment:
document.getElementById why do you need code JS to get a value in a React app? @PuniaMohit